src/collection.js

```diff
@@ -297,11 +297,11 @@ export default class Collection {
    * @param  {Object} [options.headers] The headers object option.
    * @return {Promise<Object, Error>}
    */
-  getRecord(id, options = {}) {
+  async getRecord(id, options = {}) {
     const path = endpoint("record", this.bucket.name, this.name, id);
     const reqOptions = this._collOptions(options);
     const request = { ...reqOptions, path };
-    return this.client.execute(request);
+    return await this.client.execute(request);
```

Comment:
Thanks for wondering, that made me look into eslint docs and discover that `return await` is [always implicit](http://eslint.org/docs/rules/no-return-await), therefore unnecessary.